project_proposal/solutions.md

```diff
@@ -0,0 +1,17 @@
+### Solutions
+Our distributed system consists of five sub-systems, where we are writing three
+of them.
+
+
+#### OPC Client
```

Comment:
OPC-UA: I think it's important to add the 'UA' part.

Reply:
Changed